### PR TITLE
Feature/yes no pattern

### PR DESCRIPTION
**Pull request by katedee**
Added the yes/no pattern to the Address page, and the MaritalStatus page
And apparently since we're doing this pattern, the edit pages are no longer necessary

- updated address and martialStatus to have a post
- updated their schemas and tests accordingly
- remove cypress edits run, since we were only editing on those two pages
- update the cypress "simple" run through to reflect the addition of the yes/no buttons

image
image

### CODE REVIEW

File: routes/personal/personal.spec.js [hunk 1]
```diff
@@ -109,88 +77,11 @@ describe('Test /personal responses', () => {
...
-        const response = await request(app)
-          .post('/personal/address/edit')
-          .send(badRequest.send)
-        const $ = cheerio.load(response.text)
-        expect($(badRequest.firstErrorId).attr('autofocus')).toEqual('autofocus')
-        expect(response.statusCode).toBe(422)
-      })
-    })
-
-    test('it returns a 302 with valid address', async () => {
```
Comment (pcraig3) [inline routes/personal/personal.spec.js]: 🧨

File: views/personal/address.pug [hunk 2]
```diff
@@ -25,7 +26,15 @@ block content
                 div #{data.personal.address.line1}
               div #{data.personal.address.city}, #{data.personal.address.province}
               div #{data.personal.address.postalCode}
+  
+  form.cra-form(method='post')
+    +yesNoRadios('confirmAddress', null, 'Is this information correct?', errors)
+      .no-info#noInfo
+        .no-info__wrapper
+          p #{__('If this information doesn’t match your records, you can’t change it here.')}
+          p #{__('You should not use this service to file information you believe to be incorrect.')}
```
Comment (pcraig3) [inline views/personal/address.pug]: Looks pretty copy+paste but we're doing a ✨ content audit ✨soon so we can change if needed.

File: views/personal/address-edit.pug [hunk 3]
```diff
@@ -1,54 +0,0 @@
...
-block content
-
-  h1 #{title}
-
-  div
-    p #{__('Please enter the correct mailing address information below. If a letter is sent to this address, you must be able to receive it.')}
-
-  div
-    form.cra-form.cra-form--lg(method='post')
-      h2 #{__('Your mailing address')}
```
Comment (pcraig3) [inline views/personal/address-edit.pug]: This was my favourite form in the whole app 😿

File: views/personal/address.pug (hunk 2 above)
Comment (katedee) [reply views/personal/address.pug]: It is, but there are no mocks for this content, so it's just generic right now

File: views/personal/address-edit.pug (hunk 3 above)
Comment (katedee) [reply views/personal/address-edit.pug]: I know, I thought about making the PR description just a long apology to you

**Pull request merged**